pkg/cli/conn.go

```diff
@@ -53,6 +53,17 @@ func ControlSocketName(_ CVD, cs ConnStatus) string {
 	return fmt.Sprintf("%d.sock", cs.ADB.Port)
 }
 
+func EnsureConnDirsExist(controlDir string) error {
```

Comment:
nit: just `dir` here?

Reply:
there is more than one: the control sockets directory and the logs directory, the latter inside the former